e2e/playwright/sketch-tests.spec.ts

```diff
@@ -1883,7 +1883,7 @@ profile004 = circleThreePoint(sketch001, p1 = [13.44, -6.8], p2 = [13.39, -2.07]
       })
     }
   )
-  test(
+  test.fixme(
```

Comment:
This one is definitely unhappy on main and a few branches I tried, see https://github.com/KittyCAD/modeling-app/actions/runs/13330664352/job/37248442003